apps/frontend/src/app/5_pages/EmailDuplicateVerifiedPage/EmailDuplicateVerifiedPage.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { t } from 'i18next';
+
+import { EmailContentRenderer } from '../../2_molecules/EmailContentRenderer/EmailContentRenderer';
+import { translations } from '../../../locales/i18n';
+
+export const EmailDuplicateVerifiedPage = () => {
+  return (
+    <EmailContentRenderer
+      metaTitle={t(
+        translations.emailNotifications.duplicateVerifiedPage.meta.title,
+      )}
+      title={t(translations.emailNotifications.duplicateVerifiedPage.title)}
+      subtitle={t(
+        translations.emailNotifications.duplicateVerifiedPage.subtitle,
+      )}
+    />
+  );
+};
```

Comment:
```suggestion
export const EmailDuplicateVerifiedPage = () => (
  <EmailContentRenderer
    metaTitle={t(
      translations.emailNotifications.duplicateVerifiedPage.meta.title,
    )}
    title={t(translations.emailNotifications.duplicateVerifiedPage.title)}
    subtitle={t(translations.emailNotifications.duplicateVerifiedPage.subtitle)}
  />
);
```
Unnecessary return, do the same change for all the email verification state pages.